src/race_tests.cc

```diff
@@ -189,6 +189,7 @@ TEST(Race, Client) {
     EXPECT_GT(dis->Recv(fd, buf.data(), buf.size(), 0), 0);
     EXPECT_EQ(buf.substr(9, 6), "200 OK");
 
+    freeaddrinfo(result);
```

Comment:
Do this right after Getaddrinfo.
(If this value would be needed somewhere later, you'd use a unique_ptr with freeaddrinfo as custom deleter.)